stats/views.py

```diff
@@ -43,15 +47,40 @@ def get_north_star(self):
             "percent": percent,
         }
 
+    def get_votes_histogram(self):
+        # YES, this performs one COUNT query per day in a range of 30 days
+        # (so 30 queries), but I think this is just fine for now.
+        labels = []
+        totals = []
+        positives = []
+        for date in self.get_date_range():
+            labels.append(date.strftime("%Y-%m-%d"))
+            votes = Vote.objects.filter(created_at__lt=date)
+            totals.append(len(votes))
+            positives.append(len([1 for v in votes if v.value == 1]))
+        return {
+            "labels": list(reversed(labels)),
+            "totals": list(reversed(totals)),
+            "positives": list(reversed(positives)),
+        }
+
+    def get_date_range(self):
+        base = timezone.now()
+        return [base - datetime.timedelta(days=x) for x in range(30)]
+
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         erp_qs = Erp.objects.published()
+        votes_data = self.get_votes_histogram()
 
         context["north_star"] = self.get_north_star()
         context["nb_filled_erps"] = erp_qs.count()
         context["communes"] = Commune.objects.erp_stats()[:10]
         context["nb_contributors"] = self.get_nb_contributors()
         context["top_contributors"] = self.get_top_contributors()
         context["top_voters"] = self.get_top_voters()
+        context["votes_histogram_labels"] = json.dumps(votes_data["labels"])
+        context["votes_histogram_totals"] = json.dumps(votes_data["totals"])
+        context["votes_histogram_positives"] = json.dumps(votes_data["positives"])
```

Comment:
I think there's an `escapejs` template tag we could use directly in the template.